frontend/config-assistant/src/components/gui-editor/properties/StringProperty.vue

```diff
@@ -1,19 +1,28 @@
 <script setup lang="ts">
-import {ref} from 'vue';
+import {computed, ref} from 'vue';
 import InputText from 'primevue/inputtext';
 import type {PathElement} from '@/model/path';
 import {watchThrottled} from '@vueuse/core';
+import type {JsonSchema} from '@/helpers/schema/JsonSchema';
 
 const props = defineProps<{
   propertyName: PathElement;
   propertyData: string | undefined;
+  propertySchema: JsonSchema;
 }>();
 
 const emit = defineEmits<{
   (e: 'update_property_value', newValue: string | undefined): void;
 }>();
 
 const valueProperty = ref<string | undefined>(props.propertyData);
+
+const placeHolderValue = computed(() => {
```

Comment:
Do not copy paste the logic in every component, make it a function in a corresponding TS file and reuse it